src/app/utils/util-types.ts

```diff
@@ -34,6 +34,14 @@ export function isIn<O extends Record<string | number, any>>(key: keyof O, obj:
 
 /** performs `Object.keys()` but properly narrows `key` */
 /*@__INLINE__*/
-export function objectKeys<O extends Record<string | number, any>>(obj: O) {
+export function objectKeys<O extends Record<string | number, unknown>>(obj: O) {
   return Object.keys(obj) as (keyof O)[];
 }
```

Comment:
I know this was added a while back but I don't understand why we have this, since object keys are always strings and this can return `number[]` in some cases...

Reply:
Yeah it works with the fiction that object keys can be numbers, and also handles the case where we have a lookup object that's keyed by an enum - `objectKeys` will then return an array that's typed with the enum's type.

Reply:
Well, the only usage right now seems to be the Records coreSettings code and that one doesn't even type correctly, so I don't see the value in this util function just yet.